examples/aio-sandbox/main.py

```diff
@@ -0,0 +1,16 @@
+from agent_sandbox import Sandbox
+
+# Initialize client
+client = Sandbox(base_url="http://localhost:8080")
+home_dir = client.sandbox.get_context().home_dir
+
+# Execute shell commands
+result = client.shell.exec_command(command="ls -la")
+print(result.data.output)
+
+# File operations
```

Comment:
The `run_command` calls are missing timeouts. If a command hangs, the script will be stuck indefinitely. Consider adding a `timeout` parameter to these calls to make the script more resilient, e.g., `client.run_command("ls -l /", timeout=60)`.